### PR TITLE
[MIG] edi_account_oca: Migration to 15.0

### PR DESCRIPTION
**Pull request by ao-landoo**
Standard migration

### CODE REVIEW

File: edi_account_oca/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Creu Blanca
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Edi Account",
+    "summary": """
+        Define EDI Configuration for Account Moves""",
+    "version": "15.0.1.0.0",
+    "license": "LGPL-3",
+    "author": "Creu Blanca,Odoo Community Association (OCA)",
```
Comment (etobella) [inline edi_account_oca/__manifest__.py:10]: Can you add the following here:
```
    "maintainers": ["etobella"],
    "development_status": "Beta",
```

**Review by etobella — APPROVED** [review]
Standard migration. No major changes.

Can you append my proposed changes please :smile:

File: edi_account_oca/__manifest__.py (hunk 1 above)
Comment (ao-landoo) [reply edi_account_oca/__manifest__.py:10]: Done

**Review by manuelcalerosolis — APPROVED** [review]

**Review by albariera — APPROVED** [review]
LGTM :+1:

**Pull request merged**